src/components/Error.jsx

```diff
@@ -96,6 +96,7 @@ const addLogAttachmentCommentToReportURL = (reportURL) => {
 const addOSPrettyNameCommentToReportURL = (reportURL, prettyName) => {
     const newUrl = new URL(reportURL);
     const comment = newUrl.searchParams.get("comment") || "";
+    // eslint-disable-next-line quotes
```

Comment:
This was valid comment, single quotes used with _(..) method results in non translated content. 

Also, we probably do not want this text translated, bugzilla it read by us and we want the reports in english.

I know I have some translated content above, but that text it prompting the user to attach some log files, so we need to communicate to the reporter in their own language. This information you add is for us.

Reply:
I would I think have a new section created for this purpose:

---[ System & Environment Information ]---
OS: put the pretty name here


In follow up commit we can also add the anaconda-webui and anaconda versions there.